homeassistant/components/homeassistant/scene.py

```diff
@@ -139,7 +162,24 @@ def _convert_states(states):
 
     async def create_service(call):
         """Create a scene."""
-        scene_config = SCENECONFIG(call.data[CONF_SCENE_ID], call.data[CONF_ENTITIES])
+        snapshot = call.data[CONF_SNAPSHOT]
+        entities = call.data[CONF_ENTITIES]
+
+        for entity_id in snapshot:
+            state = hass.states.get(entity_id)
+            if state is None:
+                _LOGGER.warning(
+                    "Entity %s does not exist and therefore cannot be snapshot",
```

Comment:
I looked it up and it should be `snapshotted`. Sorry.

https://en.m.wiktionary.org/wiki/snapshotted

Reply:
No worries. I went the same way and wasn't completely sure in the end :smile:

Reply:
Done